Dalamud/Data/LuminaExtensions/LanguageExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using Lumina.Data;
+
+namespace Dalamud.Data.LuminaExtensions
+{
+    /// <summary>
+    /// Extensions to <see cref="Language"/>.
+    /// </summary>
+    public static class LanguageExtensions
```

Comment:
Can you make this extend ClientLanguage instead of Lumina's Language? The language of the icon, if they do have languages, should be automatically chosen based on StartInfo.Language, like Excel.

Reply:
That should do the trick !